client/driver/executor/checks_test.go

```diff
@@ -0,0 +1,51 @@
+package executor
+
+import (
+	"strings"
+	"testing"
+
+	docker "github.com/fsouza/go-dockerclient"
+)
+
+// dockerIsConnected checks to see if a docker daemon is available (local or remote)
+func dockerIsConnected(t *testing.T) bool {
```

Comment:
Can you create a helper package that this and the driver rely on to share these functions